modules/persistent-storage-csi-group-snapshots-create.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-group-snapshots.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-group-snapshots-create_{context}"]
+= Creating a volume group snapshot
+
+When you create a `VolumeGroupSnapshot` object, {product-title} creates a volume group snapshot.
+
+.Prerequisites
+* Logged in to a running {product-title} cluster.
+* A persistent volume claim (PVC) created using a CSI driver that supports `VolumeGroupSnapshot` objects.
```

Comment:
Since we actually create the PVC group in the procedure, I changed to wording to indicate that you should create the PVCs that you want to group for the snapshot.